modules/analyzing-the-collector-pod-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: PROCEDURE
+[id="analyzing-the-collector-pod-status_{context}"]
+= Analyzing the Collector pod status
+
+Examining the pod’s most recent status is another easy way to determine the cause of a Collector crash. Failure messages are recorded to the most recent status and are accessible using the `kubectl` or `oc` command:
+
+[source,terminal]
+----
+# Substitute your collector pod into this command
+$ kubectl describe pod -n stackrox collector-vclg5
+----
+
+Example output:
+
+[source,terminal]
+----
+[...]
+    Last State:     Terminated
+      Reason:       Error
+      Message:      No suitable kernel object downloaded
+      Exit Code:    1
+      Started:      Fri, 21 Oct 2022 11:50:56 +0100
+      Finished:     Fri, 21 Oct 2022 11:51:25 +0100
+[...]
+----
+
+In this example, you can see that the collector has failed to download a kernel driver.
```

Comment:
**nitpick:** Maybe we can explain this by using code comments as done in the examples I've mentioned in [another comment](https://github.com/openshift/openshift-docs/pull/54763/files#r1080907276).

Reply:
ack